content/webapp/pages/article-series.tsx

```diff
@@ -147,8 +162,25 @@ const ArticleSeriesPage: FC<Props> = props => {
         contributors={series.contributors}
         seasons={series.seasons}
       >
-        {articles.length > 0 && (
-          <SearchResults items={series.items} showPosition={true} />
+        <SearchResults items={series.items} showPosition={true} />
+        {articles.totalPages > 1 && (
```

Comment:
This is to prevent these buttons appearing on series with <20 articles (i.e. most of them!).